### PR TITLE
ubuntu 14 => 18 in instructions

### PR DESCRIPTION
**Pull request by dgrove-oss**
@aviweit  -- does this capture our off-list discussion on getting this to work in 18.04?

### CODE REVIEW

**Comment by aviweit** [comment]
@dgrove-oss, yes, it seems to be fine from my side. I noticed you added npm, which seems to be fine.

Thanks.

File: tools/ubuntu-setup/misc.sh [hunk 1]
```diff
@@ -22,7 +22,7 @@ set -x
 export DEBIAN_FRONTEND=noninteractive
 
 sudo apt-get update -y
-sudo apt-get install -y ntp git zip tzdata lsb-release
+sudo apt-get install -y ntp git zip tzdata lsb-release npm
```
Comment (rabbah) [inline tools/ubuntu-setup/misc.sh]: fwiw, I've had to also install `unzip` for the digital ocean droplets.

Comment (dgrove-oss) [reply tools/ubuntu-setup/misc.sh]: ok, let's add `unzip` to the apt-get as well.

Comment (dgrove-oss) [reply tools/ubuntu-setup/misc.sh]: done.

**Pull request merged**